src/commands/build.ts

```diff
@@ -253,26 +254,28 @@ export async function command(commandOptions: CommandOptions) {
 
     messageBus.emit('WORKER_UPDATE', {id, state: ['RUNNING', 'yellow']});
     for (const [dirDisk, dirDest, allFiles] of includeFileSets) {
-      for (const fileLoc of allFiles) {
-        const fileExtension = path.extname(fileLoc).substr(1);
-        if (!match.includes(fileExtension)) {
+      for (const locOnDisk of allFiles) {
+        const inputExt = getExt(locOnDisk);
+        if (
+          !match.includes(inputExt.baseExt.substr(1)) &&
+          !match.includes(inputExt.expandedExt.substr(1))
+        ) {
```

Comment:
This is one dangerous change from adding `.`s to srcFileExtensionMapping. We’re still using `substr(1)` to more generously match extensions in `build:*` scripts.

We’re also testing matches for `expandedExt` just to prepare for the additional ability to add a build script to `.module.css` in addition to `.css`, but that functionality doesn’t go beyond this check, for now.

Reply:
+1. You could also solve at the config level, basically have `workerConfig.match` === `['.js']` instead of `['js']` internally so that the commands can always assume a `.` exists when comparing extensions.

Reply:
Ooh good idea! I like that.